### DISCUSSION TITLE
Fresh install cannot finish - right denied core_coresettings table

### DISCUSSION THREAD

**Discussion by nightwolf-1** [amidaware/tacticalrmm#1720] [General]
**Server Info (please complete the following information):**
 - OS: [Debian 12]
 - Browser: [nothing]
 - RMM Version (nothing because the installation failed):

**Installation Method:**
  - [ x] Standard
  - [ ] Docker

**Describe the bug**
when I install tactical rmm with a self-signed certificate (./install --insecure) the script gets stuck with this message (Mesh Central not ready yet...). 

When I ran the troubleshooting script, it told me that nats, nats-api and mesh were not running. I then used the systemctl status command to check what was happening. I noticed that the configuration file (nats-rmm.conf) didn't exist, so I ran the following command, which solved the problem (/rmm/api/env/bin/python /rmm/api/tacticalrmm/manage.py reload_nats) and I was able to start nats. As for nats-api, it just wasn't started.

Then I looked at mesh and did the following command as indicated in the documentation (/rmm/api/env/bin/python /rmm/api/tacticalrmm/manage.py check_mesh). This is where it gets tricky: I get the error i put in attachement file
[tactical_rmm_django_error.txt](https://github.com/amidaware/tacticalrmm/files/13788650/tactical_rmm_django_error.txt)

**To Reproduce**
1. Launch the install script with --insecure
2. Do the configuration 
3. See this message "Mesh Central not ready yet..."

**Expected behavior**
The result of this command (/rmm/api/env/bin/python /rmm/api/tacticalrmm/manage.py check_mesh) on the documentation

**Additional context**
I just want to install tactical rmm. I just have ssh on the server.
It is a fresh install

**Comment by wh1te909** [comment]
troubleshooting and all the other stuff you did is only for after install, not during. wipe the vm, run the install script again and look for any errors, there probably would have been errors before you got to the "meshcentral not ready yet". if you don't see any errors and still get stuck, then leave the script running, open another terminal and run `sudo systemctl status --full --no-pager meshcentral` so we can see the error and why it's stuck

**Comment by wh1te909** [comment]
found the issue, for some reason in the last few days, something with https://github.com/nodesource/distributions changed and installing nodejs 18 on Debian is no longer installing `npm`, no idea if bug or wtf they did. Anyway I just pushed a [fix](https://github.com/amidaware/tacticalrmm/commit/ccc9e44ace1b2049394dbb2ebbf31a3d063288fd) for this (just switched to Node 20 since its working on that version) so go ahead and wipe your vm, download the latest install script and try it again, will work.

**Reply by nightwolf-1** [reply]
Lors de l'installation "apt" disait que nmp étais un paquet recommandé mais je n'y ai pas prêté plus d'attention que cela.

Merci beaucoup pour votre aide. Je vais essayer et si ça marche je reviendrais vers vous pour vous le dire

**Reply by nightwolf-1** [reply]
I retry the installation from scratch and now it work thank you so much for your help.

I'ts probably a bug but who knows
